tests/unit/samplegen/test_template.py

```diff
@@ -301,6 +312,226 @@ def test_dispatch_map_loop():
     )
 
 
+def test_print_input_params():
+    check_template(
+        '''
+        {% import "feature_fragments.j2" as frags %}
+        {{ frags.printInputParams([{"base": "squid",
+                                    "body": [{"field": "mass",
+                                              "value": "10 kg",
+                                              "input_parameter": "mass"},
+                                             {"field": "length",
+                                              "value": "20 m",
+                                              "input_parameter": "length"}]},
+                                    {"base": "clam",
+                                     "body": [{"field": "diameter",
+                                               "value": "10 cm"}]},
+                                    {"base": "whelk",
+                                     "body": [{"field": "color",
+                                               "value": "red",
+                                               "input_parameter": "color"}]},
+                                    ]) }}
+        ''',
+        "\nmass, length, color"
+    )
+
+
+def test_render_calling_form_request():
+    check_template(
+        '''
+        {% import "feature_fragments.j2" as frags %}
+        {% set callingFormEnum  = {
```

Comment:
Is there any way you can use `utils.CallingForm` here and below? That way you're checking against the actual enum values being used.

Reply:
My initial thought was "technically doable but tricky," but I started messing with it and found a neat solution.